### DISCUSSION TITLE
Example with map styling & language options

### DISCUSSION THREAD

**Discussion by andrisi** [inocan-group/vue3-google-map#228] [Q&A] [answered]
Could you add an example where the map is styled with an array inside the options.styles? Or the cloud based styling. I can't seem to get it working, seems like the options don't get through. Same for map language. Thanks a lot.

**Comment by HusamElbashir** [comment, marked as answer]
Hi @andrisi 

See this example:
https://codesandbox.io/s/vue3-google-map-demo-forked-7v7t4s?file=/src/App.vue

Note that you'll need an API key for the styling to work

**Comment by andrisi** [comment]
thanks! @HusamIbrahim
